File: app/src/main/java/com/skx/tomike/javabean/CatalogItem.java
package com.skx.tomike.javabean;

/**
 * Created by shiguotao on 2016/8/1.
 * <p>
 * 目录条目
 */
public class CatalogItem {
    private String name;
    private String value;

    public CatalogItem(String name, String value) {
        this.name = name;
        this.value = value;
    }

    public String getName() {
        if (name == null) {
            name = "";
        }
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getValue() {
        if (value == null) {
            value = "";
        }
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }

}

File: app/src/main/java/com/skx/tomike/catalog/CatalogItem.kt
package com.skx.tomike.catalog

/**
 * Created by shiguotao on 2016/8/1.
 *
 * 目录条目
 */
class CatalogItem constructor(
        // 条目名称
        val name: String,
        // route 路径
        val path: String = "",
        // 值，取目标activity 的class，没啥特殊作用，就是为了方便点击查看的，实际项目中不可。
        val value: String?
) {
    constructor (name: String, path: String) : this(name, path, "") {

    }
}